model/stats/db.go

```diff
@@ -786,6 +796,34 @@ func (filter StatsFilter) buildMatchStageForTask() bson.M {
 		DbTestStatsIdProjectKeyFull:   filter.Project,
 		DbTestStatsIdRequesterKeyFull: bson.M{"$in": filter.Requesters},
 	}
+
+	// After cutting over the below build variants to use resmoke spawned
```

Comment:
Can you add a TODO to remove this once the data has been deleted by the TTL?

Reply:
Done.